package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "gpu.js": "^2.10.4",
-    "thaw.js": "^2.1.0"
+    "thaw": "^2.0.1"
```

Comment:
We are using this package: 
https://www.npmjs.com/package/thaw.js

Reply:
You're right, I messed up the plugins